Extensions/DebuggerTools/debuggertools.ts

```diff
@@ -12,6 +12,24 @@ namespace gdjs {
       export const pause = function (runtimeScene: gdjs.RuntimeScene) {
         runtimeScene.getGame().pause(true);
       };
+
+      /**
+       * Draw collisions hitboxes and points
+       * @param runtimeScene - The current scene.
+       */
+      export const drawCollisionsAndPoints = function (
+        runtimeScene: gdjs.RuntimeScene,
+        viewInsivibleInstance: boolean,
+        viewCustomPoints: boolean,
+        viewPointsNames: boolean
+      ) {
+        runtimeScene.renderCollisionsAndPoints(
+          true,
```

Comment:
Do you think we should have a parameter to set it to false?

Reply:
how we call it?
enableDebugDrawRenderedObjectsPoints ? ^^

Reply:
Its worth to make it an action parameters?
If we want enable/disable the action we can just delete it or disable the event.

Reply:
Besides it is not possible to activate or deactivate a boolean option of an action via events.
I don't really see the point.